### PR TITLE
Create issue_template.md

### PR DESCRIPTION
**Pull request by vozersky**

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+<!--- Provide a link to a live example or a clear set of steps to reproduce the issue-->
+1.
+2.
+3.
+4.
+
+### Expected behavior
+<!--- Tell us what should happen -->
+
+### Actual behaviour
```
Comment (Mizzick) [inline issue_template.md]: It could be useful to write something about the logs.

**Comment by Mizzick** [comment]
How is it supposed to be used?

**Comment by ameshkov** [comment]
@Mizzick 
> How is it supposed to be used?

Check it here:
https://github.com/AdguardTeam/AdguardForWindows/issues/new

**Comment by Mizzick** [comment]
Akhhh, nice!

**Pull request merged**